kubernetes/services/api-gateway.yaml

```diff
@@ -15,7 +15,8 @@ spec:
         app: apimanager
     spec:
       containers:
-        - image: zhonghaolyu/repo:apiKube4
+        #- image: zhonghaolyu/repo:apiKube4
+        - image: fwnetworking/controller:apiKube1
```

Comment:
As discussed in teams, we want to use the docker image tag to track the image version, maybe that would come later?

Reply:
@kevin-zhonghao I believe you made the change. Could you push your change please?

Reply:
NP, I will do it right now ~